docs/topics/dynamic-content.rst

```diff
@@ -206,7 +206,7 @@ data from it:
     >>> import lxml.etree
     >>> from parsel import Selector
     >>> javascript = response.css('script::text').get()
-    >>> xml = lxml.etree.tostring(js2xml.parse(javascript), encoding='unicode')
+    >>> xml = lxml.etree.tostring(js2xml.parse(javascript), encoding='utf-8')
```

Comment:
if I'm not mistaken, the return type would be different; encoding='unicode' makes output unicode, while encoding='utf8' makes output utf8-encoded string (i.e. bytes).

Reply:
Yes, this is correct.